Javascript/Goodboy-Oldboy /script.js

```diff
@@ -0,0 +1,18 @@
+let dogYearBirth = 2014;
+let dogYearFuture = 2031;
+let shouldShowResultInDogYears = true;
+
+let dogAge;
+if (shouldShowResultInDogYears) {
+  dogAge = dogYearFuture - dogYearBirth;
```

Comment:
This line is repeated in both cases - true and false. What if you create a variable outside the `if else` statement, and you write here directly the `console.log`s? Then you can avoid repeating `dogYearFuture - dogYearBirth` two times, and having two separate `if else` statements.